style.css

```diff
@@ -2,7 +2,6 @@ body{
     font-family: "DDG_ProximaNova","DDG_ProximaNova_UI_0","DDG_ProximaNova_UI_1","DDG_ProximaNova_UI_2","DDG_ProximaNova_UI_3","DDG_ProximaNova_UI_4","DDG_ProximaNova_UI_5","DDG_ProximaNova_UI_6","Proxima Nova","Helvetica Neue","Helvetica","Segoe UI","Nimbus Sans L","Liberation Sans","Open Sans",FreeSans,Arial,sans-serif;
     color: #222;
     line-height: 1.6;
-   
```

Comment:
??? Why did you change body style?

Reply:
If we dont delete that `overflow:hidden`, we can only see the screen that show the top html element. I had to delete that so that we can see all our works in that looooong page, so we can scroll down the page.